src/seedu/addressbook/data/person/Module.java

```diff
@@ -0,0 +1,63 @@
+package seedu.addressbook.data.person;
+
+import seedu.addressbook.data.exception.IllegalValueException;
+
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Represents a Person's module in the address book.
+ * No validation yet.
+ */
+public class Module {
+
+    public static final String EXAMPLE = "CS2130";
+    public static final String MESSAGE_MODULE_CONSTRAINTS = "Module names should be spaces, numbers or alphabetic characters";
+    public final String fullModule;
```

Comment:
Class variables should never be declared public according to the coding standard